### PR TITLE
Move up to Python 3.6

### PR DESCRIPTION
**Pull request by cclauss**

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py27,py34,py35
+envlist = py27,py34,py35.py36
```
Comment (chadwhitacre) [inline tox.ini]: Need a comma instead of a dot, ya?

Comment (cclauss) [reply tox.ini]: LOL...  YES!!

**Comment by cclauss** [comment]
Still no joy.

**Comment by cclauss** [comment]
It is beyond me.

**Pull request closed without merging**